Sources/Core/API/Actor.swift

```diff
@@ -45,7 +45,12 @@ public final class Actor: InstanceHashable, ActionPerformer, Activatable, Movabl
     /// The rotation of the actor along the z axis.
     public var rotation = Metric() { didSet { layer.rotation = rotation } }
     /// The scale of the actor. Does not affect its size, rect or collision detection.
-    public var scale: Metric = 1 { didSet { layer.scale = scale } }
+    public var scale: Metric = 1 {
+        didSet {
+            layer.scale = scale
```

Comment:
Could you use the same pattern here as for `position` and `size`, that is put it on a single line (as to not push down the implementation too much), and to delegate to a `scaleDidChange(from:)` method, where you first check if the scale indeed changed and then do the layer update + `rectDidChange()` call? Sorry for the nitpicky details 😅

Reply:
ah yeah, that sounds great, not nitpicky at all.